hosting/logistics/agenda_template.qmd

```diff
@@ -0,0 +1,49 @@
+---
+title: Agenda template
+---
+
+
+This is a draft agenda based on the experiences from the nextGEMS hackathons.
+
+
+## Draft agenda
+- break times (catering) are exemplary and might change during the planning process; 
+- beginning and end of each day depends also on room availability
+
+| Monday |  | room(s) |
+| -------- | -------- | --- |
+| 12:00 -- 15:00 | preparation     | 
+| 13:00 -- 15:00 | registration | e.g. foyer |
+| 15:00 -- 18:00 | welcome session | large auditorium |
+| evening | potential ice breaker event | foyer/off-site |
+| | | |
+| **Tuesday** |  |  |
+| 09:00 -- 20:00 | group work   | group rooms |
+| | | |
+| 11:00 -- 11:30 | coffee break | e.g. foyer |
```

Comment:
Teams should probably plan for talks that introduce the technical set-ups and workflows.
My impression is that this was less necessary for the later nextGEMS hackathons, as more and more people were used to DKRZ.

On the other hand, the pre-hackathon online sessions also did a great job in that direction. That would also be worth mentioning somewhere.